packages/grid/src/Grid.css

```diff
@@ -143,6 +143,10 @@
   --grid-rowSeparator-color: var(--uitkGrid-rowSeparator-color, var(--grid-rowSeparator-color-default));
   --grid-columnSeparator-color: var(--uitkGrid-columnSeparator-color, var(--grid-columnSeparator-color-default));
   --grid-rowSeparator-color-divided: var(--uitkGrid-rowSeparator-color-divided, var(--grid-rowSeparator-color-divided-default));
+
+  --grid-columnGhost-borderColor: var(--uitkGrid-columnGhost-borderColor, var(--uitk-palette-interact-border-hover));
```

Comment:
does --uitk-selectable-borderColor-hover work here?

Reply:
Darrin is working on characteristics for grids. Not sure if selectable is the best option here.